src/config/index.ts

```diff
@@ -0,0 +1,25 @@
+import ethereumDev from './ethereum/development';
+import ethereumProd from './ethereum/production';
+
+const configs = {
```

Comment:
I would probably move this whole config folder out of `src` folder and put one level up in root

Reply:
This is what I did at first, but since the absolute path starts at `src` it makes it much easier to use in the app : 
```typescript
import config from 'config';
```

I can try to add config as alias in the tsconfig.json.

Reply:
no need, its fine.